redaxo/src/addons/structure/plugins/content/package.yml

```diff
@@ -1,5 +1,5 @@
 package: structure/content
-version: '2.6.0'
+version: '2.6.1'
```

Comment:
Sollte man nicht besser die Versionsnummer erst zu einem erscheinenden Release erhöhen?

Reply:
Am besten fände ich es so:
- Wenn man einen "Bugfix" macht, schauen ob die Version bereits ein `-dev`-Suffix enthält. Wenn ja, nichts machen, wenn nein die hinterste Stelle um eins erhöhen und `-dev` dran hängen (z.B. `2.6.0` -> `2.6.1-dev`)
- Wenn man ein "Feature" erstellt, genauso vorgehen, aber dafür sorgen, dass die letzte Stelle `0` ist, somit ggf. die mittler erhöhen (z.B. `2.6.1` -> `2.7.0-dev`)

Aber ich denke meist nicht dran in den PRs und erhöhe sie daher erst beim Release. Was auch nicht so dramatisch ist. Aber etwas besser fände ich es schon, wenn die Version eine `-dev`-Version bereits im Repo wäre, wenn sich was am Addon geändert hat.

Zusätzlich ist noch bei Plugin-Versionen Folgendes zu beachten: Diese setze ich niemals höher, als die Addonversion, sondern würde dann die Addonversion entsprechend auch hochsetzen.
Es kann aber vorkommen, dass die Addonversion erhöht wird, die Pluginversion aber stehen bleibt, wenn sich im Plugin nichts geändert hat.
So ist in 5.6.1 das structure-Addon 2.6.1 enthalten, das content-Plugin aber noch als 2.6.0.
Somit müsste es hier aber nun mindestens `2.6.2-dev` sein, und im Addon dann auch, da es structure 2.6.1 schon gibt.

Reply:
Ich war hier etwas zwiegespalten. Da aber der Effekt der Änderung sonst nicht nachvollziehbar schien habe ich es reingenommen.

Reply:
mir wäre es am liebsten diese dinge werden kurz vorm release gemacht... alles andere zwischendrinn ist in meinen augen verwirrend und viele werden es dann auch nicht "richtig" machen.

Reply:
Das ist auch meine Befürchtung, dass man das nicht einheitlich hinbekommt, daher ist es für mich auch absolut ok, es einheitlich erst vor dem release zu machen.
(Der Nachteil ist halt, falls jemand die Github-version nutzt, steht dort noch die alte Version, obwohl sich was geändert hat. Aber damit wird man wohl leben können! ;) )

Reply:
in der whoops error page zeigen wir ja schon die git revision an.

ggf. macht es sinn diese info auch ausserhalb von error pages zu haben, oder darauf zu reagieren?

Reply:
Wobei es ja auch vorkommen kann, dass sich jemand nur ein addon aus der git-version holt, weil da ein bugfix drin ist oder so. hier geht es ja gerade vor allem auch um die addonversionen, weniger um die core-version.

Reply:
daher wäre doch ein `x.x.x-dev` nicht schlecht oder? Dann wüsste man auch direkt, dass man eine Git-Version von dem Addon/Plugin hat.